provider/data_source_datacenter.go

```diff
@@ -0,0 +1,28 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceDataCenter() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceDataCenterRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the Nelson datacenter",
+			},
+			"datacenter_url": {
```

Comment:
What would an example URL be?

Reply:
I actually don't know. I found it when there when I forked the project.